doc: known issues: Add entry regarding Matter certification. (#20013)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-9-0

> KRKNWK-19804: Descriptor Device Type List does not include OTA Requestor device on endpoint 0.

Reviewer: ```suggestion
KRKNWK-19804: Descriptor Device Type List does not include OTA Requestor device on endpoint 0
```

>   Any Matter sample does not include the OTA Requestor Device Type on endpoint 0 in the Descriptor Device Type List.

Reviewer: ```suggestion
  None of the Matter samples include the OTA Requestor Device Type on endpoint 0 in the Descriptor Device Type List.
```
